build-tools/tasks/test-utils.js

```diff
@@ -22,16 +22,22 @@ function compileTypescript(theme) {
 
 function generateTestUtilsForComponents(signalCompletion) {
   const componentNamesKebabCase = listPublicItems(path.join(testUtilsSrcDir, 'dom'));
+  const publicComponents = listPublicItems('src');
 
-  const components = componentNamesKebabCase.map(testUtilsFolderName => {
-    const name = pascalCase(testUtilsFolderName);
-    const pluralName = pluralizeComponentName(name);
-    return {
-      name,
-      pluralName,
-      testUtilsFolderName,
-    };
-  });
+  const isPublicComponent = name => publicComponents.includes(name);
```

Comment:
Can we fix that by moving all internal components to under test-utils/internal.

We already have should have snapshot tests for the test utils, so if more components are added to the root folder in the future - the test will show that.

Reply:
Currently, there are no internal components in the test-utils folder except the Annotation. This PR thus does not change the test utils that are currently generated, but guards against exposing internal components similarly in the future.